modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/SwaggerTypes.java

```diff
@@ -60,9 +65,34 @@ public static String toDefinitionType(SwaggerTypeFormat dataType) {
         } else if ("boolean".equals(dataType.getType())) {
             return "boolean";
         } else {
+            if ("array".equals(dataType.getType())) {
```

Comment:
Well maybe rephrasing it is necessary. 
This method should deal with primitive types AND lists of primitive types. 
Should I change its name ?